front-end/src/components/AppNavBar.tsx

```diff
@@ -27,54 +26,96 @@ interface Props {
+  </svg>);
+    const settingsIcon = (<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" height="25" width="25" stroke="currentColor">
+    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M10.325 4.317c.426-1.756 2.924-1.756 3.35 0a1.724 1.724 0 002.573 1.066c1.543-.94 3.31.826 2.37 2.37a1.724 1.724 0 001.065 2.572c1.756.426 1.756 2.924 0 3.35a1.724 1.724 0 00-1.066 2.573c.94 1.543-.826 3.31-2.37 2.37a1.724 1.724 0 00-2.572 1.065c-.426 1.756-2.924 1.756-3.35 0a1.724 1.724 0 00-2.573-1.066c-1.543.94-3.31-.826-2.37-2.37a1.724 1.724 0 00-1.065-2.572c-1.756-.426-1.756-2.924 0-3.35a1.724 1.724 0 001.066-2.573c-.94-1.543.826-3.31 2.37-2.37.996.608 2.296.07 2.572-1.065z" />
+    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M15 12a3 3 0 11-6 0 3 3 0 016 0z" />
+</svg>);
+const loginIcon = (<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" height="25" width="25" stroke="currentColor">
+<path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M11 16l-4-4m0 0l4-4m-4 4h14m-5 4v1a3 3 0 01-3 3H6a3 3 0 01-3-3V7a3 3 0 013-3h7a3 3 0 013 3v1" />
+</svg>);
+const profileIcon = (<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" height="25" width="25" stroke="currentColor">
+<path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z" />
+</svg> );
+
   // display pages accessible to a logged in author
   function loggedIn() {
     return (
       <>
         <NavItem onClick={close}>
-          <NavLink to="#" className="nav-link" onClick={() => setIsFriendRequestOpen(!isFriendRequestOpen)}>Friend Requests</NavLink>
-          <FriendRequestListModal {...props} isFriendRequestOpen={isFriendRequestOpen} setIsFriendRequestOpen={setIsFriendRequestOpen} />
+          <NavLink to="/inbox" className="nav-link">
+            {inboxIcon} Inbox
+          </NavLink>
         </NavItem>
         <NavItem onClick={close}>
-          <NavLink to="/settings" className="nav-link">Settings</NavLink>
+          <NavLink
+            to={`/author/${props.loggedInUser?.authorId}`}
+            className="nav-link"
+          >
+            {profileIcon} Profile
+          </NavLink>
         </NavItem>
         <NavItem onClick={close}>
-          <NavLink to={`/author/${props.loggedInUser?.authorId}`} className="nav-link">Profile</NavLink>
+          <NavLink to="/settings" className="nav-link">
+            {settingsIcon} Settings
+          </NavLink>
         </NavItem>
         <NavItem onClick={close}>
-          <NavLink to="/" className="nav-link" onClick={() => logOut()}>Log Out</NavLink>
+          <NavLink to="/" className="nav-link" onClick={() => logOut()}>
+            {logoutIcon} Log Out
+          </NavLink>
```

Comment:
thinking of getting rid of the labels next to the icon and have it so that the labels only appear when you hover over the icon - might make the UI look cleaner?

Reply:
Personally, I prefer to have the labels there always, since I suspect that showing the label on hover will shift around the other labels, giving the UI a bit of a shaky effect

This is subjective though, so I trust your judgement with this stuff either way :+1:

Reply:
hmm maybe I'm not saying it right or I'm not understanding what you mean (totally get wanting the labels there though), but I was thinking of using this in particular: https://reactstrap.github.io/components/tooltips/

Reply:
I see, I though we were planning to render a <p> tag  beside the icon or something. Tooltips look nice!